packages/shared/src/graphql/posts.ts

```diff
@@ -392,15 +392,39 @@ export const VIEW_POST_MUTATION = gql`
 export const sendViewPost = (id: string): Promise<void> =>
   request(graphqlUrl, VIEW_POST_MUTATION, { id });
 
-export const getLatestChangelogPost = async (
-  loggedIn: boolean,
-): Promise<Post> => {
-  const feedData = await request<FeedData>(graphqlUrl, SOURCE_FEED_QUERY, {
-    source: 'daily_updates',
-    first: 1,
-    loggedIn,
-    ranking: RankingAlgorithm.Time,
-  });
+export const LATEST_CHANGELOG_POST_QUERY = gql`
+  query SourceFeed($source: ID!, $first: Int, $ranking: Ranking) {
+    page: sourceFeed(source: $source, first: $first, ranking: $ranking) {
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+      edges {
+        node {
+          id
+          title
+          createdAt
+          image
+          permalink
+          numComments
+          numUpvotes
+          summary
+        }
+      }
+    }
+  }
+`;
```

Comment:
@rebelchris let me know if this is good

Reply:
Looks good to me, just see my suggestion for the variables